examples/README.md

```diff
@@ -61,12 +71,12 @@ curl -X POST -H "Content-Type: application/json" -d '{
     "sandbox_name": "'$RESTORE_SANDBOX'",
     "namespace": "'$NAMESPACE'",
     "checkpoint_path": "'$CHECKPOINT_PATH'",
-    "root": "'$ROOT'"
+    "root": "'$ROOT'",
+    "is_k3s":"false"
```

Comment:
```suggestion
```
default is false and it should be removed soon iirc we did remove it from the api.